### DISCUSSION TITLE
System crash when playing osu!lazer offline?

### DISCUSSION THREAD

**Discussion by SoffiMeower** [ppy/osu#20774] [Q&A]
### Type

Crash to desktop

### Bug description

So, the internet was out temporarily at my house, and, when attempting to play osu!lazer offline, (besides from the stupidly long beatmap loading times when offline), it also seemingly crashed my system 24 seconds into the song. I barely got into the pause menu when the GPU hanged, leaving me with the pause-sound seemingly playing normally, the Keyboard wouldn't change on Numlock or Capslock, and Alt-Tab, Ctrl-Shift-Esc, Ctrl-Alt-Delete did nothing. The disk activity light was flickering a lot, but when I rebooted, WhoCrashed had no crash log.

### Version

2022.1008.2-lazer

### Logs

[updater.log](https://github.com/ppy/osu/files/9792692/updater.log)
[database.log](https://github.com/ppy/osu/files/9792693/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9792694/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9792695/network.log)
[runtime.log](https://github.com/ppy/osu/files/9792696/runtime.log)
[performance.log](https://github.com/ppy/osu/files/9792697/performance.log)

**Comment by bdach** [comment]
There's no direct crash log in the log file. This may indicate a driver-level failure.

Please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.

**Comment by SoffiMeower** [comment]
@bdach Here's a discord download for it since Github is so picky with file uploads...
https://cdn.discordapp.com/attachments/1026379840707891231/1030896626929254430/clickcirclesmylad.evtx

**Reply by bdach** [reply]
No osu! crash here, but a lot of noise by `Salad.Bowl.Service.exe`, which seems to be a glorified crypto miner.

If you left that running while the game was running then I wouldn't be surprised if your system just ran out of resources and started aggressively swapping, which would lead to something like what you describe in the OP.

To spell it out clearly: don't run crypto miners when also playing video games.

**Reply by SoffiMeower** [reply]
> No osu! crash here, but a lot of noise by `Salad.Bowl.Service.exe`, which seems to be a glorified crypto miner.
> 
> If you left that running while the game was running then I wouldn't be surprised if your system just ran out of resources and started aggressively swapping, which would lead to something like what you describe in the OP.
> 
> To spell it out clearly: don't run crypto miners when also playing video games.

I didn't have Salad running while playing the game. ...Eh, I'll uninstall it, haven't used it in a while anyways .w.
